source/code_of_conduct.html.haml

```diff
@@ -0,0 +1,29 @@
+.row
```

Comment:
Since we aren't using bootstrap these divs aren't going to do much.  Can you make this page match our others?  Here's an example: https://github.com/eriedayofcode/site/blob/master/source/speakers.html.haml

We'll also need a `page 'code-of-conduct.html'` in `config.rb`

Can we rename this file to `code-of-conduct` so that the slug pattern will automatically match up?